src/gobupload/apply/main.py

```diff
@@ -111,7 +112,7 @@ def apply(msg):
 
         # Track eventId before event application
         entity_max_eventid, last_eventid = get_event_ids(storage)
-        before = min(last_eventid, before or last_eventid)
+        before = min(entity_max_eventid or 0, before or sys.maxsize)
 
```

Comment:
Well, it's only for logging purposes, but anyway:
If there are no events and no entities before should be None instead of 0.
The same holds for after.
The current expression is indeed wrong, it always worked because there were always events to apply.
But, the error only happens if apply is called for new entities and empty events. Quite strange situation. Importing an empty collection.
Why change the code and report the entity_max_eventid instead of the last_eventid?

Reply:
I see, the code now reports the minimum event nr of the collection before and the maximum event nr of the collection afterwards. Normally the last number should be equal to the last_eventid if everything has gone well.